tendrl/gluster_integration/flows/brick_mapping/__init__.py

```diff
@@ -0,0 +1,22 @@
+from tendrl.commons import flows
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+# This flow will be invoked only after cluster is available
+# in the tendrl (either by creation/import)
+class BrickMapping(flows.BaseFlow):
```

Comment:
I was GetBrickMapping before @anivargi suggested this name, for better consistency in API layer